packages/app/src/routers/SavedviewsRouter/components/ComponentInputs.tsx

```diff
@@ -0,0 +1,177 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *
+ * This code is for demonstration purposes and should not be considered production ready.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ColorDef } from "@bentley/imodeljs-common";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { ColorPickerButton } from "@bentley/ui-components";
+import { Slider } from "@itwin/itwinui-react";
+import React, { useState } from "react";
+
+import "./ComponentInputs.scss";
+import { generateJSONInStyles } from "./GenerateJSONStyles";
+import { reloadDisplayStyle } from "./ReloadDisplayStyle";
+
+export interface ComponentsColorPickerProps {
+  label: string;
+  path: any[];
+  pathLength: number;
+  dataType?: string;
+}
+export function ComponentsColorPicker({
+  label,
+  path,
+  pathLength,
+  dataType,
+}: ComponentsColorPickerProps) {
+  return (
+    <div className="label-with-color-picker">
+      <h3 className="text-setting">{label}</h3>
+      <ColorPickerButton
+        className="color-picker-height text-setting"
+        initialColor={ColorDef.fromString("skyblue")}
+        onColorPick={(color: ColorDef) => {
+          const newBackground = IModelApp.viewManager.selectedView?.displayStyle
+            .clone()
+            .toJSON();
+
+          if (pathLength !== path.length) {
+            path.pop();
+          }
+          if (dataType === "JSON") {
+            path.push(color.toJSON());
+          }
+          if (dataType === "RGB") {
+            path.push(color.getRgb());
+          }
+
+          if (newBackground !== undefined) {
+            generateJSONInStyles(newBackground, path);
+          }
+
+          if (newBackground !== undefined) {
+            generateJSONInStyles(newBackground, path);
+          }
+
+          // Load display style again
+          const vp = IModelApp.viewManager.selectedView;
+          if (vp !== undefined && newBackground !== undefined) {
+            reloadDisplayStyle(vp, newBackground);
+          }
+        }}
+      />
+    </div>
+  );
+}
+
+export interface ComponentsSliderProps {
+  label: string;
+  path: any[];
+  pathLength: number;
+  min: number;
+  max: number;
+  step: number;
+}
+export function ComponentsSlider({
+  label,
+  path,
+  pathLength,
+  min,
+  max,
+  step,
+}: ComponentsSliderProps) {
+  const [value, setValue] = useState(0.2);
+  return (
+    <div className="label-with-slider text-setting">
```

Comment:
This component requires a newer version of itwinui-react, which has a different itwinui-css dependency and it screws up with the formatting. So I told Brian to avoid using that for now and just use a formatted div in this case. That ok @raplemie?

Reply:
I discovered this issue as well working on the official portal, in order for the CSS not to be broken too much, you can use yarn resolutions in the package.json: 
```
    "@itwin/itwinui-css": "0.58.0",
    "@itwin/itwinui-react": "1.38.1"
``` 
  This should ensure that every components of the portal will use the same css, it will screw up some of the other components (the tiles will look weird) but that's ok, I'll fix them later, but it will be easier than rebuilding the whole thing.

Try it at least once, if it does not work, its ok.

Reply:
I tried changing the code and rebuilding the node_module, but it still has the same issue

Reply:
When using resolutions, you need to completely remove the node_modules in your tree otherwise the old stuff is stuck there
`npx rimraf .\node_modules .\packages\app\node_modules`
should get you a "fresh start".

Reply:
I removed the three node_modules by hand and did yarn install, then yarn start. Would that do the same thing?

Reply:
Yes, ok, too bad if that doesnt work, dont waste any more time on this, use your previous solution.